Yoast/wordpress-seo
# Add local for woocommerce to license page

## Summary

This PR can be summarized in the following changelog entry:

* Added the Local SEO for WooCommerce extension to the licenses page.

## Relevant technical choices:

* I've used the USPs from the Yoast extension page. I hope this is okay.

## Test instructions

This PR can be tested by following these steps:

* Checkout this branch and have Local SEO for WooCommerce installed and activated. You should only see that extension box on the page. The Local SEO box should be hidden.
* Deactivate Local SEO for WooCommerce and see the Local SEO box be visible again.

Fixes #8335
